bot/src/HexBug/resources/lang/zh-CN/main.ftl

```diff
@@ -0,0 +1,217 @@
+# command/group/parameter name:
+#       |------------------------------|
+
+# command/group/parameter description:
+#       |--------------------------------------------------------------------------------------------------|
+
+# common parameters
+
+-parameter_visibility =
+        visibility
```

Comment:
Command/group/parameter names do apparently support localization: https://discord.com/developers/docs/interactions/application-commands#localization
It's up to you if you want to translate them or leave them in English. The linked page has an example showing them being translated, but I don't know what it'll look like in Discord. If you want, you could just translate them for one of the command, then I can merge it so you can see what it looks like in the Discord UI and decide if you want them translated or not.